ansible_collections/arista/avd/docs/porting-guides/4.x.x.md

```diff
@@ -15,6 +15,16 @@ built in to `eos_designs`.
 
 ### Fabric variables
 
+#### MLAG peer link allowed vlans
+
+`mlag_peer_link_allowed_vlans` no longer has a default value of "2-4094".
+
+To retain AVD 3.8 behavior set `mlag_peer_link_allowed_vlans` to "2-4094":
+
+```yaml
+mlag_peer_link_allowed_vlans: "2-4094"
```

Comment:
This example is misleading by not presenting that the var is set under the node type
```suggestion
l3leaf:
  defaults:
    mlag_peer_link_allowed_vlans: "2-4094"
```